index.js

```diff
@@ -4,7 +4,7 @@ const supportsHyperlinks = require('supports-hyperlinks');
 
 module.exports = (text, url, options = {}) => {
 	if (!supportsHyperlinks.stdout) {
-		return options.fallback ? options.fallback(text, url) : `${text} (${url})`;
+		return options.fallback ? options.fallback(text, url) : `${text} ( ${url} )`;
```

Comment:
Use zero-width space character (using unicode escape syntax), so it still visually looks like the parens are still closed around it.

Reply:
Nice idea, done :)